README.md

```diff
@@ -1,17 +1,51 @@
-# Title <!-- Replace 'Title' and this comment with a title or name of your algorithm -->
-Description <!-- Replace this entire line with a short description of your algorithm -->
+# Greenness Transformer
+Provides a variety of calculations using RGB indices in order to monitor crop/plant
+growth and health, taking into account variations between lighting and cameras
 
 ## Authors
-<!-- Replace this comment with a list of the authors of the algorithm pointing out who the main contacts are -->
+Chris Schnaufer, University of Arizona, Tucson, AZ
+Jacob van der Leeuw: University of Arizona, Tucson, AZ
+
+## Overview
+The Greenness Transformer is meant to provide several calculations on image rgb data in order to get an idea of plant growth and
+health. A more in-depth explanation can be found at [this link](https://docs.google.com/document/d/1cAm5w1Bs6dB1SHgf-HVmwwbebLhbZMUvTkXLtas_-xI/edit)
+
+## Algorithm Description
+This transformer provides calculations using the red, green, and blue values for pixels in an image ranging from calculating the
+percentage of an image which is green to normalized difference calculations and more
 
 ## References
```

Comment:
Can you adjust the headings below to provide clarity as to what goes with what. Having everything at the same level makes it difficult to read. Any approach that clarifies the relationships between the text is fine.